### PR TITLE
Mega 1

### PR DESCRIPTION
**Pull request by mrkoloev**
поменял в урлах уфлрусь на мега лигу

### CODE REVIEW

File: daddyleagues/bot.py [hunk 1]
```diff
@@ -63,7 +65,7 @@ def week_num(message, cmd):
 @app.route('/res')
 def res(message):
     chat_dest = message['chat']['id']
-    url = 'http://www.daddyleagues.com/uflrus/gamerecap/548877212'
+    url = f'http://www.daddyleagues.com/{league_name}/gamerecap/548877212'
```
Comment (wicked-kiD) [inline daddyleagues/bot.py:68]: Why don't we use https?

**Review by wicked-kiD — CHANGES_REQUESTED** [review]

File: daddyleagues/bot.py [hunk 2]
```diff
@@ -21,6 +21,8 @@
 key = ''
 #273770462
 
+league_name = 'mega'
```
Comment (wicked-kiD) [inline daddyleagues/bot.py:24]: I believe it will be better to have variable with `league_url`

Comment (mrkoloev) [reply daddyleagues/bot.py:24]: @wicked-kiD mr. Vertykalka, i believe we should create separate config file

File: daddyleagues/bot.py (hunk 1 above)
Comment (lespauI) [reply daddyleagues/bot.py:68]: Daddyleague where only http those times

**Pull request merged**